ql/models/model.hpp

```diff
@@ -121,7 +124,8 @@ namespace QuantLib {
         virtual void generateArguments() {}
         std::vector<Parameter> arguments_;
         boost::shared_ptr<Constraint> constraint_;
-        EndCriteria::Type shortRateEndCriteria_;
+        EndCriteria::Type endCriteria_;
```

Comment:
Ok, I am fine with that.

> Am 24.02.2016 um 10:25 schrieb Luigi Ballabio <notifications@github.com>:
> 
> In ql/models/model.hpp <https://github.com/lballabio/QuantLib/pull/55#discussion_r53912390>:
> 
> > @@ -121,7 +124,8 @@ namespace QuantLib {
> >          virtual void generateArguments() {}
> >          std::vector<Parameter> arguments_;
> >          boost::shared_ptr<Constraint> constraint_;
> > -        EndCriteria::Type shortRateEndCriteria_;
> > +        EndCriteria::Type endCriteria_;
> Ok about the values, but I'd keep the current name for shortRateEndCriteria_ even though we all agree that it's unfortunate. It's a protected data member, so it could be used in user code.
> 
> —
> Reply to this email directly or view it on GitHub <https://github.com/lballabio/QuantLib/pull/55/files#r53912390>.
>